.github/workflows/test.yaml

```diff
@@ -0,0 +1,102 @@
+name: Test Recipes
+
+on:
+  workflow_dispatch:
+      inputs:
+        version:
+          description: 'Radius version number to use (e.g. 0.1.0, 0.1.0-rc1, edge). Defaults to edge.'
+          required: false
+          default: 'edge'
+          type: string
+  push:
+    branches:
+      - v*.*
+      - edge
+    paths:
+      - "local-dev/**"
+      - ".github/workflows/**"
+  pull_request:
+    types: [opened, synchronize, reopened]
+env:
+  RUN_IDENTIFIER: recipestest-${{ github.run_id }}-${{ github.run_attempt }}
+  APP_NAME: local-dev-recipe-app
+  APP_NAMESPACE: local-dev-recipe-app
+jobs:
+  test:
```

Comment:
If this is not supposed to run on forks please add the if check like this: https://github.com/radius-project/radius/blob/main/.github/workflows/assets.yaml#L29.

Reply:
Should this check be added?

Reply:
I was thinking we should run this on fork as well, wondering if there is reason why we dont want to run on forks?